lib/Dialect/AIEVec/Utils/Utils.cpp

```diff
@@ -105,4 +106,15 @@ VectorType getFlattenedVectorType(VectorType vecTy) {
       vecTy.getElementType());
 }
 
+std::optional<xilinx::AIE::AIEArch> getAIEVersionFromModule(ModuleOp moduleOp) {
```

Comment:
The downside of this is that it makes the AIEVec lowerings useless outside of a device.   Perhaps they should *also* allow passing from the command line?

Reply:
Makes great sense to me. I realized that those changes I made to the pass are not necessary to make the workflow work. Those commits have been reverted.